storage/FastSeparateChainingHashTable.hpp

```diff
@@ -145,10 +145,10 @@ class FastSeparateChainingHashTable
         header_->buckets_allocated.load(std::memory_order_relaxed);
     void *bucket_ptr = static_cast<char *>(buckets_) + kValueOffset;
     for (std::size_t bucket_num = 0; bucket_num < num_buckets; ++bucket_num) {
-      for (std::size_t handle_num = 0; handle_num < handles_.size(); ++handle_num) {
+      for (std::size_t handle_id = 0; handle_id < num_handles_; ++handle_id) {
         void *value_internal_ptr =
```

Comment:
I think it's okay either way. If we are using `char *`, then we need `reinterpret_cast` to cast `char *` to `std::uint8_t *`. Here two `static_cast`'s should semantically be better.